lib/index.js

```diff
@@ -50,9 +50,11 @@ page.onError = function (err, trace) {
 	].join('\n'));
 };
 
-page.onResourceReceived = function () {
-	page.injectJs(opts.es5shim);
-};
+if (opts.es5shim){
+  page.onResourceReceived = function () {
+    page.injectJs(opts.es5shim);
+  };	
```

Comment:
The indentation here is wrong. The rest of the repo uses tabs, this is two spaces.